devassistant/command_runners.py

```diff
@@ -929,29 +936,29 @@ def _strip_dir_prefix(cls, prefix, path):
 
 @register_command_runner
 class AsUserCommandRunner(CommandRunner):
+
     @classmethod
     def matches(cls, c):
         return c.comm_type.startswith('as ')
 
     @classmethod
-    def get_user_from_comm_type(cls, comm_type):
-        split_type = comm_type.split()
+    def get_user_from_comm_type(cls, c):
```

Comment:
This method is named `..._from_comm_type` and it accepted comm_type before, now it accepts the command and it gets the type out of it, I'd suggest either keep original behavior, or rename the method to `get_user_from_command`